hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -275,19 +281,19 @@ export default function HackathonDetails(props: IProps) {
       {hackathonData.startDate &&
         !isHackathonLoading &&
         !isHackathonError &&
-        type === 'header' && (
+        type === Enum.Header && (
           <div>
             <h2>Title: {hackathonData.title}</h2>
             <h2>
-              Start Date: {new Date(hackathonData.startDate).toDateString()}
-              End Date: {new Date(hackathonData.endDate).toDateString()}
+              Start Date: {new Date(hackathonData.startDate).toDateString()} End
+              Date: {new Date(hackathonData.endDate).toDateString()}
             </h2>
             <h2>All Ideas ({hackathonData.ideas?.length})</h2>
 
             <IdeaCardList
               ideas={relevantIdeaList}
               columnSize={6}
-              type={'Archive'}
+              type={Enum.Archive}
```

Comment:
is `archive` the right choice here? These are the header details, archive sounds wrong.  I would also think that whatever the value is should actually come from the IdeaCardList file